### ISSUE TITLE
Enable https

### ISSUE THREAD

**Issue by aslakr** [asciimath/asciimath.github.io#26]
It seems [custom domains on GitHub Pages now supports https](https://github.blog/2018-05-01-github-pages-custom-domains-https/). Would it be possible to enable this for asciimath.org

**Comment by drlippman** [comment]
It looks like this may require some DNS reconfiguration.  Who owns the domain and has access to make DNS changes?

**Comment by ad-si** [comment]
I think @pkra has access to the domain, right?

**Comment by pkra** [comment]
> I think @pkra has access to the domain, right?

Not in a long time.

**Comment by drlippman** [comment]
I'm getting warnings that the DNS records for the site need to be updated.  Who owns the domain??  Maybe @ad-si ?

**Comment by ad-si** [comment]
No, I don't. It's registered since 2014 at NameCheap. Does that ring a bell for anyone? Maybe @jipsen?

**Comment by jipsen** [comment]
Sorry, I have never heard of NameCheap, and I’m fairly sure I wasn't involved in setting up the asciimath.org<http://asciimath.org> domain name or the site.

Are there maybe some old github messages that solve this mystery? (If not, I can try to dig up some old emails from 2014, but this could take a while…)

—Peter

On Jan 27, 2021, at 3:36 PM, Adrian Sieber <notifications@github.com<mailto:notifications@github.com>> wrote:

External Message



No, I don't. It's registered since 2014 at NameCheap. Does that ring a bell for anyone? Maybe @jipsen<https://nam03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fjipsen&data=04%7C01%7Cjipsen%40chapman.edu%7C8c19834a3da3456b31f708d8c31c6858%7C809929af2d2545bf9837089eb9cfbd01%7C1%7C0%7C637473874065313261%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=vMxoufvDm4fK%2BgYoQ9i9t5w25%2BM1oqAMN7z7JLIREp8%3D&reserved=0>?

—
You are receiving this because you were mentioned.
Reply to this email directly, view it on GitHub<https://nam03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fasciimath%2Fasciimath.github.io%2Fissues%2F26%23issuecomment-768649871&data=04%7C01%7Cjipsen%40chapman.edu%7C8c19834a3da3456b31f708d8c31c6858%7C809929af2d2545bf9837089eb9cfbd01%7C1%7C0%7C637473874065313261%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=lVz6ja3vOaQ28IuCk%2FJxecQu7Egg%2FLjLfIL6pvilf0E%3D&reserved=0>, or unsubscribe<https://nam03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fgithub.com%2Fnotifications%2Funsubscribe-auth%2FAAP5CDBGYJXPFZ4MORHVI5LS4CPQZANCNFSM4KCL3Z5A&data=04%7C01%7Cjipsen%40chapman.edu%7C8c19834a3da3456b31f708d8c31c6858%7C809929af2d2545bf9837089eb9cfbd01%7C1%7C0%7C637473874065323253%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C1000&sdata=qo%2FXzm9yE03o5Ddmm4t%2FZIgUDiEnQj621%2BEwB1nLvgA%3D&reserved=0>.

NOTE: This email originated from outside Chapman’s network. Do not click links or open attachments unless you recognize the sender and know content is safe.

**Comment by drlippman** [comment]
I looked through my old emails, and @pkra was definitely the one who originally registered the domain.  Peter, do you have any idea who you transferred ownership to?  Or does the MathJax organization own it?

**Comment by pkra** [comment]
@drlippman @jipsen ping me by email, please.

**Comment by drlippman** [comment]
I was able to track it down - apparently AMS owns asciimath.org, and I got in touch with someone who can help make the needed changes.

**Comment by drlippman** [comment]
Closing as this seems to have been resolved.
